openedx/features/student_certificates/views.py

```diff
@@ -117,8 +119,24 @@ def student_certificates(request):
             'certificate_url': "%s%s" % (settings.LMS_ROOT_URL, certificate_url),
             'course_start': start_date.strftime('%b %d, %Y') if start_date else None,
             'completion_date': completion_date.strftime('%b %d, %Y') if completion_date else None,
+            'is_program_cert': False,
         })
 
+    program_credentials = get_credentials(request.user, credential_type='program')
+    for credential in program_credentials:
+        program_name = [attribute['value'] for attribute in credential.get('attributes',{})
+                        if 'name' in attribute and attribute['name'] == 'program_name']
+        if program_name:
+            user_certificates.append({
+                'course_name': program_name[0],
+                'course_title': program_name[0],
+                'social_sharing_urls': {},
+                'certificate_url': '',
+                'course_start': '',
+                'completion_date': date_from_str(credential.get('created')).strftime('%b %d, %Y'),
```

Comment:
Please make a new file constants.py and define constant for 
COMPLETION_DATE_FORMAT = '%b %d, %Y'

secondly you can define a method to convert date to string by giving string format. just like we define date_from_str.. 

We can move **date_to_str** and **date_from_str** to **philu_utils.py** so that can be used in other apps as well